src/scripts/components/DropdownMultiSelectAdvanced.jsx

```diff
@@ -49,7 +49,7 @@ class DropdownMultiSelectAdvanced extends React.Component {
   };
 
   handleToggle = () => {
-    if (this.state.isViewAllItems) {
+    if (this.state.isViewAllItems && this.state.searchText.length > 0) {
       this.setState({ searchText: '' });
       this.props.fetchAllItems('', this.props.filterName.toLowerCase());
```

Comment:
```suggestion
if (this.state.searchText.length > 0) {
      this.props.fetchAllItems('', this.props.filterName.toLowerCase());
    }
```

Reply:
@yogitattn We should clear searchText when searchText is available. Why we need to clear searchText when it is not availble.

Reply:
My suggestion is this: 
handleToggle = () => {
    if (this.state.isViewAllItems) {
      this.setState({ searchText: '' });
      if (this.state.searchText.length > 0) {
        this.props.fetchAllItems('', this.props.filterName.toLowerCase());
      }
    }
    this.setState(prevState => ({ isViewAllItems: !prevState.isViewAllItems }));
  };